### PR TITLE
Total taxed cartItem and cart price

### PR DESCRIPTION
**Pull request by InformatiqueProg**
See issue #1772

### CODE REVIEW

**Comment by InformatiqueProg** [comment]
If accepted, "mini-cart.html" in module HookCart need the same modification.

File: templates/frontOffice/default/cart.html [hunk 1]
```diff
@@ -121,7 +121,7 @@ <h3 class="name"><a href="{$PRODUCT_URL nofilter}">{$TITLE}</a></h3>
                                 </div>
                             </td>
                             <td class="subprice">
-                                <span class="price">{format_money number={$real_price * $QUANTITY} symbol={currency attr="symbol"}}</span>
+                                <span class="price">{format_money number={$real_price} symbol={currency attr="symbol"}}</span>
```
Comment (gillesbourgeat) [inline templates/frontOffice/default/cart.html]: Hello,
It's ```$real_total_price``` not ```$real_price```
Cheers,

**Comment by gillesbourgeat** [comment]
Hello,
For the methods ```getTotalRealTaxedPrice, getTotalTaxedPrice, getTotalTaxedPromoPrice```, can you add ```@since``` in the documentation of the method.
Example : 
```php
    /**
     * @since Version 2.3
     * @param Country $country
     * @return float
     */
    public function getTotalTaxedPrice(Country $country)
```
More information http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.since.pkg.html
Cheers,

File: templates/frontOffice/default/cart.html (hunk 1 above)
Comment (InformatiqueProg) [inline templates/frontOffice/default/cart.html]: i re-use $real_price, because it's no more used...
But your right, it's not clear, i'll change this

**Comment by gillesbourgeat** [comment]
Do you know rebase and squash a pull request ?

**Comment by InformatiqueProg** [comment]
Never use, but i know what they do
I 'll try...

**Comment by InformatiqueProg** [comment]
I'm sorry,
i can't do the squash, something wrong with the empty commit "merge..."
I have done a new PR #1816

**Pull request closed without merging**